modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2438,23 +2584,28 @@ For example, if you have a cluster of three nodes, each with 8 vCPUs, then a que
 
 The query in this sample request takes a parameter of type `VERTEX<person>`:
 
-[source,bash]
-----
-curl -X POST -d '{"p":{"id":"Tom","type":"person"}}' \
-"http://localhost:9000/query/social/hello"
-
-# Specify that the query run on the first replica
-curl -X POST -d -H "GSQL-REPLICA: 2" '{"p":{"id":"Tom","type":"person"}}' \
+[source.wrap,bash]
+--
+curl -X POST -d '{"p":{"id":"Tom","type":"person"}}'
```

Comment:
Still open blocks(--) for code blocks (----). Have you verified that this renders correctly?